api/graphql/schema.graphql

```diff
@@ -574,6 +576,7 @@ input UserFilterAndOrderInput {
     skills: [ID] @builder(method: "App\\Models\\User@filterBySkills")
     isGovEmployee: Boolean @scope
     orderBy: String   # This needs to be the name of the field in the database eg. first_name
+    status: PoolCandidateStatus @builder(method: "App\\Models\\User@filterByPoolCandidateStatus")
```

Comment:
Since this only makes sense as part of PoolCandidates, it should be part of UserPoolFilterInput instead. Also, I think we should make it use an array of PoolCandidateStatuses, allowing for more flexibility.